### DISCUSSION TITLE
ATProto: Account not properly bridged? - No web user found for domain

### DISCUSSION THREAD

**Discussion by theCapypara** [snarfed/bridgy-fed#1457] [Q&A] [answered]
Opening this as a discussion for now, since I'm probably just impatient :) , it has been roughly an hour since bridging, though other accounts worked within minutes.

It seems that this particular BlueSky account: https://fed.brid.gy/bsky/explorersguild.bsky.social did not bridge properly. The user profile was created, but all activity log entries 404, they also don't show the user name, just "...".
Trying to follow results in "Bad Request - No web user found for domain".

Note: I do not run that account, I just asked the owner to bridge.

**Comment by snarfed** [comment]
Hey, sorry for the trouble! Have you looked at https://fed.brid.gy/docs#troubleshooting ?

**Reply by theCapypara** [reply]
Hey! Thanks for the quick reply. Yeah. The only thing that would apply is that the account may have blocked the bridge, but I don't think so. If I'm missing anything let me know. The profile page also doesn't show any hint on what is going on. It seems like it should work.

**Reply by theCapypara** [reply, marked as answer]
Well haha, turns out I was just impatient after all (or you just fixed it), either way, it works now! thank you for this great project.
